pkg/controller/controller_serviceplan.go

```diff
@@ -62,7 +62,7 @@ func (c *controller) reconcileServicePlanKey(key string) error {
 		return err
 	}
 	pcb := pretty.NewContextBuilder(pretty.ServicePlan, namespace, name, "")
-	plan, err := c.servicePlanLister.ServicePlans(namespace).Get(key)
+	plan, err := c.servicePlanLister.ServicePlans(namespace).Get(name)
```

Comment:
can we cover that with some unit-test to do not have any regression in the future? 🙂

Reply:
Sure, I managed to test that logic. I created a ticket for other controllers that don't have this kind of test coverage: https://github.com/kubernetes-sigs/service-catalog/issues/2799.